features/rake_task/rake_task.feature

```diff
@@ -93,4 +94,7 @@ Feature: Reek can be driven through its Task
     Then it succeeds
     And it reports:
       """
+      [DEPRECATED]
+      Starting with reek 1.6 there is only one configuration file possible.
+      Please use "config_file" instead of "config_files".
```

Comment:
I’d say let’s drop it in 2.0 already, that’s exactly what major version bumps are for.